R/read_absmaps.R

```diff
@@ -56,9 +63,19 @@ read_absmap <- function(name = NULL,
   url <- paste0(base_url, name, ".rda")
 
   # download to temporary file
-  out_path <- tempfile(fileext = ".rda")
-  download.file(url,
-                destfile = out_path)
+  out_path <- file.path(export_dir, paste0(name, ".rda"))
+
+  if (!file.exists(out_path)) {
+    download.file(url,
+                  destfile = out_path)
```

Comment:
I would add `mode = "wb"`. This specifies that the file is binary; this is automatically inferred on macOS and Linux, but as I understand it isn't always automatically inferred on Windows